mautic-inc/plugin-integrations
# Add reference_id to object mapping entity, add indexes

**Jira:** https://backlog.acquia.com/browse/MAUT-2815

Add reference id.

### Testing instruction
1. Delete integrataionsBundle from `mautic_plugins` table
2. Drop the table `mautic_sync_object_mapping`
3. Get this PR
4. Run` app/console mautic:plugin:update`
5. Verify the field is there and Salesforce integration still works.
  `app/console mautic:integrations:sync Salesforce2  -e dev -vvv --start-datetime="2011-01-01 00:00:00"`
